mcrouter/standalone_options_list.h

```diff
@@ -28,6 +28,14 @@ MCROUTER_OPTION_STRING(
     no_short,
     "Name of the carbon router to use")
 
+MCROUTER_OPTION_OTHER(
+    std::vector<std::string>,
+    listenAddresses,
```

Comment:
call this `listen_addresses` to make it look similar to other flavor options (see https://github.com/facebook/mcrouter/wiki/Flavors).
you could even call this just `addresses` (as we call the next argument just `ports` and not `listen_ports`).